README.md

```diff
@@ -49,7 +49,14 @@ Additional topics for interview prep:
   <img src="images/zdCAkB3.png">
   <br/>
 </p>
-
+<p align="center">
```

Comment:
Can we also add a link to the GitHub repo?
https://github.com/daveight/ankicode

Thanks